feat(apple): Update Automatic Instrumentation (#3959)
getsentry/sentry-docs: src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

![App Start Transaction](app-start-transaction.png)

## Slow and Frozen Frames

This feature is available for iOS, tvOS, and Mac Catalyst.

> Unresponsive UI, animation hitches, or just jank annoy users and degrade the user experience. Two measurements to track this unwanted experience are slow and frozen frames. A phone or tablet typically renders with 60 frames per second (fps). The frame rate can also be higher, especially as 120 fps displays are becoming more popular. With 60 fps, every frame has 16.67 ms to render. If your app needs more than 16.67 ms for a frame, it is a slow frame. Frozen frames are UI frames that take longer than 700 ms. A smoothly running app should try to avoid both. That's why the SDK adds these two measurements for the transactions you capture. The detail view of the transaction displays the slow, frozen, and total frames to the right.

Reviewer: "just jank" may not translate well, suggest either removing it or choosing another phrase. Otherwise, a few edits to better introduce what we're adding:
```suggestion
Unresponsive UI, animation hitches, or just jank annoy users and degrade the user experience. Two measurements to track this unwanted experience are slow frames and frozen frames. A smoothly running app should try to avoid both. That's why the SDK adds these two measurements for the transactions you capture.

- **Slow frames** are captured by the SDK when the app needs more than 16.67 ms for a frame. Typically, a phone or tablet renders with 60 frames per second (fps), though the frame rate can be higher, especially as 120 fps displays become popular. With 60 fps, every frame has 16.67 ms to render; slower than that, and the app has slow frames.
- **Frozen frames** are UI frames that take longer than 700 ms.  

The detail view of the transaction displays the slow, frozen, and total frames:
```

![Slow and Frozen Frames](slow-frozen-frames.png)

## HTTP Instrumentation

The SentrySDK adds spans for outgoing HTTP requests to ongoing transactions that are bound to the Scope.
The SDK attaches auto-generated transactions automatically to the Scope if there is no other transaction bound.
When starting a transaction, you can use bindToScope to indicate whether the SDK should bind the new transaction to the Scope or not.

> ## How to opt out

Reviewer: ```suggestion
## Opt Out
```
